src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SymmetricKeyWrap.cs

```diff
@@ -23,7 +23,7 @@ internal static class SymmetricKeyWrap
         internal static byte[] TripleDESKeyWrapEncrypt(byte[] rgbKey, byte[] rgbWrappedKeyData)
         {
             // checksum the key
-            SHA1CryptoServiceProvider sha = new SHA1CryptoServiceProvider();
+            var sha = SHA1.Create();
```

Comment:
These Disposable objects hold native resources, so we should `using` them to avoid finalizations.